scripts/cc-cli/src/utils/bond.ts

```diff
@@ -41,7 +41,7 @@ export function parseRewardDestination(
     rewardDestinationRaw.charAt(0).toUpperCase() +
     rewardDestinationRaw.slice(1).toLowerCase();
 
-  if (rewardDestination !== "Staked" || "Stash" || "Controller") {
+  if (rewardDestination !== "Staked" && rewardDestination !== "Stash" && rewardDestination !== "Controller") {
```

Comment:
@ESPINS can you run `yarn format` and force-push.

Reply:
@atodorov  It's done.